### PR TITLE
chore: Add new localization languages

### PR DESCRIPTION
**Pull request by borisprimer**
# Description

[CHKT-3905
](https://primerapi.atlassian.net/browse/ACC-3905)

Added languages for GYG.

### CODE REVIEW

File: phrase_config.yml [hunk 1]
```diff
@@ -231,6 +231,69 @@
...
+        encoding: UTF-16
+
+    - file: Sources/PrimerSDK/Resources/Localizable/sr-RS.lproj/Localizable.strings
+      params:
+        file_format: strings
+        locale_id: Serbian
+        tags: ios-sdk
+        encoding: UTF-16
+
+    - file: Sources/PrimerSDK/Resources/Localizable/sl.lproj/Localizable.strings
```
Comment (semirp) [inline phrase_config.yml:269]: [question]: should this be `sl-SI`?

File: phrase_config.yml [hunk 2]
```diff
@@ -231,6 +231,69 @@
...
+        encoding: UTF-16
+
+    - file: Sources/PrimerSDK/Resources/Localizable/lt-LT.lproj/Localizable.strings
+      params:
+        file_format: strings
+        locale_id: Lithuanian
+        tags: ios-sdk
+        encoding: UTF-16
+
+    - file: Sources/PrimerSDK/Resources/Localizable/ca.lproj/Localizable.strings
```
[the remark this answers is not in the document]
Comment (borisprimer) [reply phrase_config.yml:255]: This is how it is setup in Phrase.com

File: phrase_config.yml (hunk 1 above)
Comment (borisprimer) [reply phrase_config.yml:269]: This is how it is setup in Phrase.com

**Comment by borisprimer** [comment]
@NQuinn27 can we do something about .strings being presented as binary files?

**Comment by NQuinn27** [comment]
> @NQuinn27 can we do something about .strings being presented as binary files?

I'm not sure we can - `*.strings text` in the `.gitattributes` should cover this - but it appears GitHub does not honour it :-/

**Review by NQuinn27 — APPROVED** [review]

**Pull request merged**